app/src/main/java/org/p2p/wallet/swap/ui/orca/OrcaSwapFragment.kt

```diff
@@ -133,15 +154,9 @@ class OrcaSwapFragment :
         setupAmountFractionListener()
 
         exchangeImageView.setOnClickListener { presenter.reverseTokens() }
-        swapDetails.setOnSlippageClickListener {
-            presenter.loadDataForSettings()
-        }
-        swapDetails.setOnPayFeeClickListener {
-            presenter.loadDataForSettings()
-        }
-        swapDetails.setOnTransactionFeeClickListener {
-            presenter.onFeeLimitsClicked()
-        }
+        swapDetails.setOnSlippageClickListener { presenter.loadDataForSettings() }
```

Comment:
swapDetails.apply

Reply:
i don't want to create too much indentation, sorry